internal/buildscripts/packaging/installer/install.ps1

```diff
@@ -535,6 +554,42 @@ if ($with_fluentd) {
     echo "- Started"
 }
 
+if ($with_dotnet_instrumentation) {
+    echo "Installing SignalFx Instrumentation for .NET ..."
+    $api = "https://api.github.com/repos/signalfx/signalfx-dotnet-tracing/releases/latest"
+    $module_name = "Splunk.SignalFx.DotNet.psm1"
+    echo "Downloading .NET Instrumentation installer ..."
+    $download = (Invoke-WebRequest $api | ConvertFrom-Json).assets | Where-Object { $_.name -like $module_name } | Select-Object -Property browser_download_url,name
+    $dotnet_auto_path = Join-Path $tempdir $download.name
+    Invoke-WebRequest -Uri $download.browser_download_url -OutFile $dotnet_auto_path
+    Import-Module $dotnet_auto_path
+    echo "Installing SignalFx Dotnet Auto Instrumentation..."
+    Install-SignalFxDotnet
+
+    echo "Setting environment variables for instrumentation ..."
+    update_registry -path "$regkey" -name "COR_ENABLE_PROFILING" -value "1"
+    update_registry -path "$regkey" -name "COR_PROFILER" -value "{B4C89B0F-9908-4F73-9F59-0D77C5A06874}"
+    update_registry -path "$regkey" -name "CORECLR_ENABLE_PROFILING" -value "1"
+    update_registry -path "$regkey" -name "CORECLR_PROFILER" -value "{B4C89B0F-9908-4F73-9F59-0D77C5A06874}"
+
+    if ($instrumentation_exclude_processes -ne "") {
+      echo "Setting SIGNALFX_PROFILER_EXCLUDE_PROCESSES environment variable to $instrumentation_exclude_processes ..."
+      update_registry -path "$regkey" -name "SIGNALFX_PROFILER_EXCLUDE_PROCESSES" -value "$instrumentation_exclude_processes"
+    }
+
+    if ($signalfx_service_name -ne "") {
+        echo "Setting SIGNALFX_SERVICE_NAME environment variable to $signalfx_service_name ..."
```

Comment:
I still have misgivings in offering service name when we are targeting IIS apps. If there is more than one on the box it doesn't work well. I would rather remove service name from the wizard.

Reply:
although here is the script not the wizard 🤦‍♂️

Reply:
Makes sense. I've removed this override.